packages/republik-crowdfundings/graphql/resolvers/_mutations/submitPledge.js

```diff
@@ -289,7 +289,10 @@ module.exports = async (_, args, context) => {
         }) // try to load existing user by email
         if (
           user &&
-          !!(await transaction.public.pledges.findFirst({ userId: user.id }))
+          !!(await transaction.public.pledges.findFirst({
+            userId: user.id,
+            'status !=': 'DRAFT',
+          }))
```

Comment:
Since only row count matters, `count()` is probably enough, and also helps not transferring unused data but a count number.

```suggestion
          !!(await transaction.public.pledges.count({
            userId: user.id,
            'status !=': 'DRAFT',
          }))
```